src/Azos/Data/EntityId.cs

```diff
@@ -69,12 +74,12 @@ public EntityId(Atom sys, Atom type, string address)
                                                 Type.GetDistributedStableHash() ^
                                                 ShardKey.ForString(Address);
 
-    public override bool Equals(object obj) => obj is EntityId other ? this.Equals(other) : false;
+    public override bool Equals(object obj) => obj is EntityId other ? Equals(other) : false;
 
     public bool Equals(EntityId other)
-      => this.Type == other.Type &&
-         this.System == other.System &&
-         this.Address.EqualsOrdSenseCase(other.Address);
+      => Type == other.Type &&
+         System == other.System &&
```

Comment:
The use of "this" makes sense for clarity in tight code blocks that compare objects. I would keep `this.` prefix in Equals implementations